### DISCUSSION TITLE
Allow to grab episode from feeds before their air date

### DISCUSSION THREAD

**Discussion by papampi** [pymedusa/Medusa#9818] [Ideas & Feature requests]
Some episodes available for streams sooner than their scheduled air time, would be a good idea to allow medusa grab them from providers or rss feeds if they show up early.
Example: 
Godfather of Harlem - s02e09 
The Walking Dead - s11e01 
The Walking Dead -  s11e02
Pops up today in torrent feeds instead of 8/23/2021

**Comment by p0psicles** [comment]
That's already possible.
Go to config - providers -> provider option & feel -> search delay. You can enter a negative valule. Like -24 to have it start searching 24 hours sooner.

**Reply by papampi** [reply]
Awesome didn't know we can set negative value.

**Comment by papampi** [comment]
So I set it to -24 and saved it but after I get back to the page later I noticed its showing 0.5 instead of -24, is this normal?
![Screenshot from 2021-08-22 15-09-14](https://user-images.githubusercontent.com/1108483/130352205-78ec46b7-5f31-4a50-9bd1-ffa422c46337.png)

**Comment by papampi** [comment]
Config file shows 30
```
[EZTV_720P_X265]
eztv_720p_x265 = 1
eztv_720p_x265_name = eztv-720p-x265
eztv_720p_x265_url = http://rss.ftp.sh/eztv-720p-x265-rss.xml
eztv_720p_x265_search_mode = eponly
eztv_720p_x265_search_fallback = 1
eztv_720p_x265_enable_daily = 1
eztv_720p_x265_enable_backlog = 1
eztv_720p_x265_enable_manualsearch = 1
eztv_720p_x265_enable_search_delay = 1
eztv_720p_x265_search_delay = 30
eztv_720p_x265_ratio = -1
eztv_720p_x265_minseed = 0
eztv_720p_x265_minleech = 0
eztv_720p_x265_cookies = ""
eztv_720p_x265_title_tag = title
```

**Comment by p0psicles** [comment]
Ow no. That's not good. I'll have to fix that. Just, few weeks ago I refactored the config/providers page.
